manager/src/main/java/com/redhat/service/bridge/manager/actions/connectors/SlackActionValidator.java

```diff
@@ -0,0 +1,41 @@
+package com.redhat.service.bridge.manager.actions.connectors;
+
+import java.util.Map;
+
+import javax.enterprise.context.ApplicationScoped;
+
+import com.redhat.service.bridge.actions.ActionParameterValidator;
+import com.redhat.service.bridge.actions.ValidationResult;
+import com.redhat.service.bridge.infra.models.actions.BaseAction;
+
+@ApplicationScoped
+public class SlackActionValidator implements ActionParameterValidator {
```

Comment:
@lucamolteni Did you create a JIRA issue for us to work on improving the ability to validate Action parameters?

Reply:
https://issues.redhat.com/browse/MGDOBR-202